solidity/contracts/stBTC.sol

```diff
@@ -326,17 +330,18 @@ contract stBTC is ERC4626Fees, PausableOwnable {
 
         // Mint the shares to the receiver.
         super._mint(receiver, shares);
-
-        return shares;
     }
 
     /// @dev This function proxies `mintDebt` call and provides compatibility
     ///      with Mezo IReceiptToken interface.
     function mintReceipt(address to, uint256 amount) external {
-        mintDebt(amount, to);
+        uint256 assets = mintDebt(amount, to);
+        if (assets == 0) {
+            revert ConvertedToZero(amount, assets);
+        }
```

Comment:
So do I understand it correctly that in `mintReceipt` the `ConvertedToZero` error will be triggered when user will request too low amount of stBTC in the context of a given state of contract, right?

And alike - in `burnReceipt` user can trigger `ConvertedToZero` by requesting to burn a low amount of asset.

But what if that's all that was minted by this user? They will not be able to burn their stBTC?